tests/test_bqpipeline.py

```diff
@@ -54,8 +63,24 @@ def test_create_job_config_destination(self):
             self.assertEqual(cfg.destination.project, 'testproject')
 
     def test_create_job_config_flags(self):
-        bq = BQPipeline(job_name='testjob', default_project='testproject', default_dataset='testdataset')
+        bq = BQPipeline(
+            job_name='testjob', default_project='testproject', default_dataset='testdataset')
         cfg = bq.create_job_config(batch=False, create=False, overwrite=False)
         self.assertEqual(cfg.priority, bigquery.QueryPriority.INTERACTIVE)
-        self.assertEqual(cfg.create_disposition, bigquery.job.CreateDisposition.CREATE_NEVER)
-        self.assertEqual(cfg.write_disposition, bigquery.job.WriteDisposition.WRITE_EMPTY)
+        self.assertEqual(cfg.create_disposition,
+                         bigquery.job.CreateDisposition.CREATE_NEVER)
+        self.assertEqual(cfg.write_disposition,
+                         bigquery.job.WriteDisposition.WRITE_EMPTY)
+
+
+class TestLogging(unittest.TestCase):
```

Comment:
Thanks for improving test coverage here!

Reply:
This was Brandon's doing. Thanks Brandon!